### PR TITLE
Update syntax and file structure in modules and update the build

### PR DESCRIPTION
**Pull request by dinuish94**

### CODE REVIEW

File: docs/learn/guides/integration-patterns/content-based-routing/Ballerina.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+org_name = "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/content-based-routing/Ballerina.lock]:
```suggestion
org_name = "wso2"
```

File: docs/learn/guides/integration-patterns/content-based-routing/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[project]
+org-name= "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/content-based-routing/Ballerina.toml]:
```suggestion
org-name= "wso2"
```

File: docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal [hunk 3]
```diff
@@ -1,34 +1,20 @@
 import ballerina/http;
-import ballerina/io;
+// import ballerina/io;
 import ballerina/test;
-import ballerina/log;
-import ballerina/mime;
+// import ballerina/log;
+// import ballerina/mime;
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal]: We can remove these commented out imports. Lines 2, 4 and 5.

File: docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal [hunk 4]
```diff
@@ -101,10 +87,4 @@ json payload3 = { "Name": "Smart Automobile" };
...
         test:assertFail(msg = "Failed to call the endpoint:");
     }
-}
-
-function stopService() {
-    test:stopServices("company_recruitment_agency_service");
-    test:stopServices("company_data_service");
-}
-
+}
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal]: New line

File: docs/learn/guides/integration-patterns/message-filtering/.gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+target
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/message-filtering/.gitignore]: New line

File: docs/learn/guides/integration-patterns/message-filtering/Ballerina.lock [hunk 6]
```diff
@@ -0,0 +1,4 @@
+org_name = "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/message-filtering/Ballerina.lock]:
```suggestion
org_name = "wso2"
```

File: docs/learn/guides/integration-patterns/message-filtering/Ballerina.toml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+[project]
+org-name= "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/message-filtering/Ballerina.toml]:
```suggestion
org-name= "wso2"
```

File: docs/learn/guides/integration-patterns/pass-through-messaging/.gitignore [hunk 8]
```diff
@@ -0,0 +1 @@
+target
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/pass-through-messaging/.gitignore]: New line

File: docs/learn/guides/integration-patterns/pass-through-messaging/Ballerina.lock [hunk 9]
```diff
@@ -0,0 +1,4 @@
+org_name = "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/pass-through-messaging/Ballerina.lock]:
```suggestion
org_name = "wso2"
```

File: docs/learn/guides/integration-patterns/pass-through-messaging/Ballerina.toml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+[project]
+org-name= "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/pass-through-messaging/Ballerina.toml]:
```suggestion
org-name= "wso2"
```

File: docs/learn/guides/integration-patterns/scatter-gather-messaging/.gitignore [hunk 11]
```diff
@@ -0,0 +1 @@
+target
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/scatter-gather-messaging/.gitignore]: New line

File: docs/learn/guides/integration-patterns/scatter-gather-messaging/Ballerina.lock [hunk 12]
```diff
@@ -0,0 +1,4 @@
+org_name = "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/scatter-gather-messaging/Ballerina.lock]:
```suggestion
org_name = "wso2"
```

File: docs/learn/guides/integration-patterns/scatter-gather-messaging/Ballerina.toml [hunk 13]
```diff
@@ -0,0 +1,5 @@
+[project]
+org-name= "dinuksha"
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/scatter-gather-messaging/Ballerina.toml]:
```suggestion
org-name= "wso2"
```

File: docs/learn/guides/integration-patterns/content-based-routing/.gitignore [hunk 14]
```diff
@@ -0,0 +1 @@
+target
```
Comment (IrushiL) [inline docs/learn/guides/integration-patterns/content-based-routing/.gitignore]: New line

File: docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal [hunk 15]
```diff
@@ -1,34 +1,17 @@
...
 
-
 function Company_Recruitments_Agency() {
 
-// Invoking the main function
-http:Client httpEndpoint = new("http://localhost:9091");
-
 // Chck whether the server is started
-test:assertTrue(serviceStarted, msg = "Unable to start the service");
+// test:assertTrue(serviceStarted, msg = "Unable to start the service");
```
Comment (Asitha) [inline docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal]: Commented out assert?

File: docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal [hunk 16]
```diff
@@ -1,34 +1,17 @@
...
-
-
-
-function startService() {
-    serviceStarted = test:startServices("company_recruitment_agency_service");
-
-    serviceStarted = test:startServices("company_data_service");
-
-}
+// boolean serviceStarted = false;
```
Comment (Asitha) [inline docs/learn/guides/integration-patterns/content-based-routing/src/company_data_service/tests/company_recruitment_agency_service_test.bal]: Commented out code?

File: docs/learn/guides/integration-patterns/backend-for-frontend/src/message_mgt/message_mgt_service.bal [hunk 17]
```diff
@@ -112,21 +110,21 @@ service message_mgt_service on httpListener {
...
 
         // Get all Messages from map and add them to response
         int i = 0;
-        foreach var(k, v) in messageMap {
-            json messageValue = v.Message;
+        foreach var v in messageMap {
+            json messageValue = <json>v.Message;
             string messageStatus = messageValue.Status.toString();
             if (messageStatus == "Unread"){
-                messageResponse.Messages[i] = messageValue;
```
Comment (ThishaniLucas) [inline docs/learn/guides/integration-patterns/backend-for-frontend/src/message_mgt/message_mgt_service.bal]: If the variable i's usage is removed, we can remove that variable itself?

Comment (dinuish94) [reply docs/learn/guides/integration-patterns/backend-for-frontend/src/message_mgt/message_mgt_service.bal]: Method has been removed in the latest version : https://github.com/wso2/ballerina-integrator/blob/master/docs/content/src/learn/tutorials/integration-patterns-and-soa/backend-for-frontend/src/healthcare_management_service/downstream_services/message_mgt_service.bal

**Pull request merged**